### ISSUE TITLE
Update accessibility scan once upstream fixes available

### ISSUE THREAD

**Issue by ethanwhite** [datacarpentry/semester-biology#911]
We use https://github.com/pa11y/pa11y-ci for auto accessibility testing, but it is currently experiencing intermittent failures when scanning large sites like this one. They are working on this issues is known and fixes are being discussed/worked on in:

* https://github.com/pa11y/pa11y-ci/issues/128
* https://github.com/pa11y/pa11y/issues/570

Once these are solved we need to update our `pa11y` and/or `pa11y-ci` dependencies to address this issue.

Note the system still works detects issues as long as the page in question isn't in the ~1% of pages that testing fails for.

**Comment by ethanwhite** [comment]
Upstream fix is now available: https://github.com/pa11y/pa11y-ci/issues/128#event-5678874736

**Comment by ethanwhite** [comment]
Everything seems to be working except the 'comment on pull request' functionality. That should be fixed and a test run to make sure that new issues are properly flagged and then this can be closed.

**Comment by ethanwhite** [comment]
Update and appears to be working
